scripts/download_all_miz.py

```diff
@@ -0,0 +1,90 @@
+""" 
+Downloads all MIZ shapefiles from this link: https://usicecenter.gov/Products/ArchiveSearchMulti?table=DailyArcticShapefiles
+
+To run:
+pixi run python scripts/download_all_miz.py
+"""
+import os
+from datetime import datetime
+
+import requests
+from bs4 import BeautifulSoup
+
+
+def format_date(date_obj: datetime) -> str:
+    """Function to format date in MM/DD/YYYY format"""
+    return date_obj.strftime("%m/%d/%Y")
+
+
+def get_date(string: str) -> str:
+    """From file link, extract date for MIZ product"""
```

Comment:
I'm curious how this `string` looks like

Reply:
I reorganized some of the file logic and wrote a more descriptive docstring, hopefully that's a lot more clear now!